src/app/components/listing/confirm/compare-uploaded-and-system-developers.jsx

```diff
@@ -59,105 +91,98 @@ function ChplCompareUploadedAndSystemDevelopers(props) {
               <TableRow
                 className={(system.name !== uploaded.name) ? classes.differentValue : ''}
               >
-                <TableCell scope="row">Developer Name</TableCell>
+                <TableCell className={classes.sectionCell} scope="row">Developer Name</TableCell>
                 <TableCell>{ system.name }</TableCell>
                 <TableCell>{ uploaded.name }</TableCell>
               </TableRow>
               <TableRow
                 className={(system.website !== uploaded.website) ? classes.differentValue : ''}
               >
-                <TableCell scope="row">Website</TableCell>
+                <TableCell className={classes.sectionCell} scope="row"><Typography><b>Website</b></Typography></TableCell>
```

Comment:
Why does this one have the Typography & <b> tags, while the others don't? I'd expect it to be standardized